.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,36 @@
+name: Documentation
+description: Issues with documentation for one of our libraries.
+labels: [documentation, question]
+body:
+
+- type: dropdown
+  attributes:
+    label: Core Library
+    multiple: true
+    options: 
+      - "@azure/msal or msal"
+      - "@azure/msal-browser"
+      - "@azure/msal-node"
+
+- type: dropdown
+  attributes:
+    label: Wrapper Library
+    multiple: true
+    options:
+      - "@azure/msal-angular"
+      - "@azure/msal-react"
+
+- type: input
+  attributes:
+    label: Documentation Location
+    description: Please provide a link to the documentation this issue is about
+    placeholder: "docs.microsoft.com"
+  validations: 
+    required: true
+
+- type: textarea
+  attributes:
+    label: Description
+    description: "Please briefly describe your issue."
+  validations: 
+    required: true
```

Comment:
We can also give them an option to open a PR on our documentation and link it to this issue template, encouraging community contributions.

Reply:
> Approved, but wondering if we should leave an (optional) area for them to include logs? Otherwise, very exciting!

Great call, added! There's also a generic comment box at the end that people can fill in with misc. information that may not be covered in the form.

Reply:
Great idea! Added a markdown section to the top of this template suggesting PRs